faq.md

```diff
@@ -21,7 +21,7 @@ Have some spare time on your hands?
 
 Hospitals and clincs administering vaccines are our end targets.
 We can add your data to our site and help bring people to you.
-It will also relieve your databases and phone calls as we can provide this directly to your customers.
+We will also retrieve your databases, relieving phone calls as we can provide this information to your customers through interactive tools.
```

Comment:
what do we mean "we will retrieve your databases"?"

IMO i think we should just say that if a hospital  in interested in providing us with vaccine availability data to provide their customers with a web portal to quickly and easily check availability, then they should contact us.

Reply:
Hmmm. Maybe we just need to delete the sentence and replace it. My meaning here was that we could display whether or not they had vaccines and direct users to their signup/reservation website.